apps/charterafrica/src/components/StyledCard/StyledCard.js

```diff
@@ -0,0 +1,26 @@
+import { Card, styled } from "@mui/material";
+
+import { neutral } from "@/charterafrica/colors";
+
+const StyledCard = styled(Card)(({ ownerState, theme }) => ({
+  backgroundColor: "#fff",
+  transition: theme.transitions.create(["filter", "border-color"]),
+  width: "100%",
+  "&:hover": {
```

Comment:
From the designs, it looks like when hovering over the image, there should also be a similar hover effect

Reply:
@kelvinkipruto not sure I get this

Reply:
@koechkevin When hovering over the `CardMedia`, on `GuaranteeCard` there is a border that appears